scripts/nb-tester/qiskit_docs_notebook_tester/config.py

```diff
@@ -25,12 +25,30 @@
 
 # We always run the following code in the kernel before running the notebook
 PRE_EXECUTE_CODE = """\
+import logging
+import re
 # Import with underscores to avoid interfering with user-facing code.
 from matplotlib import set_loglevel as _set_mpl_loglevel
 
-
 # See https://github.com/matplotlib/matplotlib/issues/23326#issuecomment-1164772708
 _set_mpl_loglevel("critical")
+
+
+# Ignore server configuration warnings from qiskit-ibm-runtime; we can't control these and they seem to be benign
```

Comment:
Hm, could you please check in with the Qiskit Runtime team in `#qiskit-runtime-open` in Slack about these? It would be good to confirm they're aware of them. We don't want users encountering these issues. Depending on their reply, we should probably create a GH issue to track reverting this patch.